docs/Developers/Git_Basics.rst

```diff
@@ -369,6 +369,9 @@ Now, to revert the file to that revision, run the command:
 Now, after the file gets reverted back to the required revision, commit the
 changes and (force)push to the remote.
 
+If at any stage you are confused or have any issue don't close the PR or issue
```

Comment:
These sentences can be worded a bit better and they are also grammatically incorrect.
Something more like:
```
If at any stage you are confused, or have an issue, do not close your Pull Request. Instead, contact us on gitter so that we can help you resolve your problem.
```